packages/textlint-tester/test/textlint-tester-plugin.ts

```diff
@@ -0,0 +1,106 @@
+// LICENSE : MIT
+"use strict";
+
+import TextLintTester = require("../src/index");
+
+const htmlPlugin = require("textlint-plugin-html");
+const noTodoRule = require("textlint-rule-no-todo");
+const maxNumberOfLineRule = require("textlint-rule-max-number-of-lines");
+const tester = new TextLintTester();
+
+tester.run(
```

Comment:
lack of valid case

- Pass a TestConfig that has multiple `rules` only
- https://github.com/textlint/textlint/pull/491/files#r163427444

Reply:
I see. Will add such case. (I tried to salvage notification from my inbox but no avail...sorry for that)